components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -117,14 +156,40 @@ - (void)setupTabBarColors {
 
 - (nonnull NSArray *)constructExampleViewControllers {
   NSBundle *bundle = [NSBundle bundleForClass:[TabBarViewControllerExample class]];
+  MDCButtonScheme *buttonScheme = [[MDCButtonScheme alloc] init];
+  buttonScheme.colorScheme = self.colorScheme;
+  buttonScheme.typographyScheme = self.typographyScheme;
+
   TBVCSampleViewController *child1 =
       [TBVCSampleViewController sampleWithTitle:@"One" color:UIColor.redColor];
+
   UIColor *blue = [UIColor colorWithRed:0x3A / 255.f green:0x56 / 255.f blue:0xFF / 255.f alpha:1];
   TBVCSampleViewController *child2 = [TBVCSampleViewController sampleWithTitle:@"Two" color:blue];
+  __weak TabBarViewControllerExample *weakSelf = self;
+  [child2 addMDCButtonWithFrame:CGRectMake(10, 120, 300, 40)
+                   buttonScheme:buttonScheme
+                          title:@"Push and Hide Tab"
+                    actionBlock:^{
+                      TabBarViewControllerExample *strongSelf = weakSelf;
+                      TBVCSampleViewController *vc =
+                      [TBVCSampleViewController sampleWithTitle:@"Push&Hide" color:UIColor.grayColor];
+                      vc.colorScheme = self.colorScheme;
+                      vc.typographyScheme = self.typographyScheme;
```

Comment:
Please use strongSelf here and above.

Reply:
Good catch! Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.